uint16[] spot_x
uint16[] spot_y
---
float32 temperature_c
